### ISSUE TITLE
Locks quality at 480p

### ISSUE THREAD

**Issue by wcbx** [cleanlock/VideoAdBlockForTwitch#60]
This extension locks the stream quality at 480p. It's useless now.

**Comment by designgears** [comment]
Only goes to 480p during the commercial.

**Comment by immolated1** [comment]
> Only goes to 480p during the commercial.

Why?

**Comment by designgears** [comment]
> Why?

Because that's the only video stream available while they show ads.

**Comment by wcbx** [comment]
> Only goes to 480p during the commercial.

False. It locked the quality at 480p period. I changed to the TTVLOL extension and it works perfectly.

**Comment by designgears** [comment]
> False. It locked the quality at 480p period. I changed to the TTVLOL extension and it seems to work (sometimes).

With version 1.5.5 that is true, this question was asked before it was released.
